docs/usermanual.md

```diff
@@ -361,7 +363,7 @@ fields codified in hexadecimal with a fixed 4 character length, without comma se
 of this attribute, adding the "batteryType" and "percentage" fields to the entity.
 
 
-##  <a name="development"/> Development documentation
+##  Development documentation
```

Comment:
Now the index link to this subsection does not work

Reply:
Fixed - facd80d - ToC regenerated